prisma/schema.prisma

```diff
@@ -388,7 +388,7 @@ model pupil {
   matchingPriority                             Int                                            @default(0)
   lastUpdatedSettingsViaBlocker                DateTime?                                      @db.Timestamp(6)
   teacherEmailAddress                          String?                                        @db.VarChar
-  registrationSource                           pupil_registrationsource_enum                  @default(dbgenerated("0"))
+  registrationSource                           pupil_registrationsource_enum                  @default(normal)
```

Comment:
I have no idea how this got into the current state, but `dbgenerated("0")` doesn't make sense as that tries to set the enum to an integer.